phys2bids/bids.py

```diff
@@ -170,3 +172,64 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
     heurpath = os.path.join(fldr, f'{name}physio')
 
     return heurpath
+
+
+def participants_file(indir, outdir, yml, sub):
+    """[summary]
+
+    Args:
+        indir ([type]): [description]
+        outdir ([type]): [description]
+        yml ([type]): [description]
+        sub ([type]): [description]
+    """
+    LGR.info('Updating participants.tsv ...')
+    file_path = os.path.join(outdir, 'participants.tsv')
+    if not os.path.exists(file_path):
+        LGR.warning('phys2bids could not find participants.tsv')
+        # Read yaml info if file exists
+        if yml is not None and os.path.exists(os.path.join(indir, yml)):
+            LGR.info('Using yaml data to populate participants.tsv')
+            with open(os.path.join(indir, yml)) as f:
+                yaml_data = yaml.load(f, Loader=yaml.FullLoader)
+            p_id = sub
+            p_age = yaml_data['participant']['age']
+            p_sex = yaml_data['participant']['sex']
+            p_handedness = yaml_data['participant']['handedness']
+        else:
+            LGR.info('No yaml file was provided. Using phys2bids data to '
+                     'populate participants.tsv')
+            # Fill in with data from phys2bids
+            p_id = sub
+            p_age = 'n/a'
```

Comment:
These three columns are not mandatory. I would skip the addition of 'n/a' columns.

Reply:
I disagree, these are the Recommended options, which are not mandatory but are in a higher level than the optional ones

Reply:
I don't agree with this. As we discussed yesterday (I think with @vinferrer and @tsalo), phys2bids should recommend those as BIDS does.

Reply:
But we're not recommending. We're just adding empty columns.

If we want to recommend, then let's add a warning that states "Hey you should add those columns by hand!"

Reply:
I have no strong opinions about it but would like to know what @vinferrer and @tsalo think